hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -188,6 +188,7 @@ export OPERATOR_EXEC=kubectl
 export CLUSTER_TYPE=openshift
 export DEV_MODE=TRUE
 export CLUSTER_HAS_EMULATED_PF=TRUE
+export OPERATOR_LEADER_ELECTION_ENABLE=true
```

Comment:
do we want it to be true also in the k8s lane?

Reply:
IMO no. We have two lanes, and for me, it's good to have this flag tested with both values. 
The alternative might be to add other lanes with all possible combinations: `ocp|leader-elect=true`, `ocp|leader-elect=false`, `k8s|leader-elect=true`, `k8s|leader-elect=false`. But it would be a waste of resources, in my opinion.

Reply:
yep agree we don't have many CI resources to test all the combinations